http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -386,7 +412,7 @@ if [ ! -z "$JUPYTER_DOCKER_IMAGE" ] ; then
     retry 3 docker exec -u root ${JUPYTER_SERVER_NAME} ${JUPYTER_SCRIPTS}/kernel/kernelspec.sh ${JUPYTER_SCRIPTS}/kernel ${KERNELSPEC_HOME}
   fi
 
-  # Install notebook.json
+  # Install notebook.json, which is related to the locking logic
```

Comment:
Specifically, I think `notebook.json` is used to populate `Jupyter.notebook.config` in JavaScript extensions. This is used in the [edit-mode.js](https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-base/custom/edit-mode.js#L11) extension that Terra/AoU use.